backend/de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/ddl/public/functions/purchase_order_dynamics_json.sql

```diff
@@ -30,6 +36,15 @@ FROM c_order porder
          INNER JOIN ad_org org ON porder.ad_org_id = org.ad_org_id
          INNER JOIN c_bpartner partner ON porder.c_bpartner_id = partner.c_bpartner_id
          INNER JOIN c_currency c ON c.C_Currency_ID = porder.C_Currency_ID
+         LEFT JOIN LATERAL (
+    SELECT externalreference
+    FROM s_externalreference
+    WHERE record_id = partner.c_bpartner_id
+      AND type = 'BPartner'
+      AND referenced_record_id = partner.c_bpartner_id
```

Comment:
referenced_record_id shall be the same as record_id, is that right?
(maybe it's a stupid question)